src/Storage.gs

```diff
@@ -39,27 +39,44 @@ function Storage_(prefix, properties, optCache) {
  * @type {number}
  * @private
  */
-Storage_.CACHE_EXPIRATION_TIME_SECONDS = 12*3600; // 12 hours.
+Storage_.CACHE_EXPIRATION_TIME_SECONDS = 21600; // 6 hours.
```

Comment:
@ljay79 
This is the latest version of Storage from the google codebase
This addition means it caches NULL's in memory rather than repeatedly hitting the properties service to get the NULL again
That may explain some of the high numbers (but low users) in https://github.com/ljay79/jira-tools/issues/177

Seems like this is all thats needed for now?